client/state/plugins/marketplace/actions.ts

```diff
@@ -40,3 +67,168 @@ export function setIsPluginInstalledDuringPurchase(
 		isPluginInstalledDuringPurchase,
 	};
 }
+
+export function resetPurchaseFlow(): { type: string } {
```

Comment:
`{ AnyAction } from 'redux'`? In general, can we type the action creators a bit better?

Reply:
So to clarify, Using AnyAction is fine as well right? Feel like redux gets a bit too verbose with typed actions.

Reply:
Yep, in any case, IMHO it's better than duplicating the returned action object in the type 😉